src/Adapter/Attribute/AttributeDataProvider.php

```diff
@@ -84,6 +84,8 @@ public static function getAttributeIdsByGroup($id_group, $not_null = false)
      */
     public function getProductCombinations($idProduct)
     {
+        $locales = $this->container->get('prestashop.adapter.legacy.context')->getLanguages();
```

Comment:
well, also it alwas was `$id_product` not `$idProduct`, I see many changes in naming convention, not sure if for better...